libs/auth/src/java_card.ts

```diff
@@ -81,6 +98,29 @@ const VX_ADMIN_CERT_AUTHORITY_CERT = {
   OBJECT_ID: pivDataObjectId(0xf2),
 } as const;
 
+/**
+ * A rough upper bound on the size of a single data object
```

Comment:
Maybe add some context around how you came to this upper bound just in case we need to reason about it in the future?

Reply:
also, isn't the max object size 2^5 Kb?

Reply:
Yes good call!

@ben It's very specifically `2^15 - 5` bytes. I went with 30K to leave breathing room, but on second thought, it doesn't make sense to add the breathing room at this level. It makes sense to add the breathing room a level up in the total capacity variable.

Made tweaks accordingly and added lots of context in: https://github.com/votingworks/vxsuite/pull/3045/commits/8f0f81f8a079225b17a521ee73eb3e5d7e4f7f62